pkg/util/hostname.go

```diff
@@ -228,6 +228,11 @@ func GetHostnameData() (HostnameData, error) {
 	// and the hostname is one of the default ones
 	if getEC2Hostname, found := hostname.ProviderCatalog["ec2"]; found {
 		log.Debug("GetHostname trying EC2 metadata...")
+		if !ecs.IsECSInstance() && ec2.IsWindowsDefaultHostname(hostName) && !config.Datadog.GetBool("ec2_use_windows_prefix_detection") {
+			// REMOVEME: This should be removed when the default `ec2_use_windows_prefix_detection` is set to true
+			log.Warn("You may want to use the EC2 instance-id for the in-app hostname." +
+				" For more information: https://dtdg.co/ec2-use-win-prefix-detection")
```

Comment:
```suggestion
				" For more information: https://docs.datadoghq.com/ec2-use-win-prefix-detection")
```
^I created this alias for the FAQ page that will be added.